### PR TITLE
Removed old and duplicate content

### PR DESCRIPTION
**Pull request by HemantMahawar**

### CODE REVIEW

**Comment by HemantMahawar** [comment]
@maertendMSFT and @mirichmo, can both of you review and provide your feedback.

**Comment by maertendMSFT** [comment]
Looks good to me.

File: docs/limitations.md [hunk 1]
```diff
@@ -17,11 +12,10 @@
 
   - **Workaround**: To mount storage in your new region, untag the existing storage account and restart PowerShell in Cloud Shell. 
       1.	Login into [Azure Portal](https://portal.azure.com) directly (ie., without using our private preview link)
-      2.	Launch the Cloud Shell (Bash)
-          * If you have an active PowerShell session [you will need to Restart Cloud Shell](#for-a-given-user-only-one-shell-can-be-active)
+      2.	Launch the Bash shell
```
Comment (mirichmo) [inline docs/limitations.md]: How about "Bash Cloud Shell" instead. I interpret "Launch Bash shell" as an instruction to open a local Bash shell.

File: docs/limitations.md [hunk 2]
```diff
@@ -36,14 +30,13 @@
     1. Refresh the web page using the browser's refresh button and re-open the console.
     1. Close the shell IFrame (the sub-window that appears after clicking on the shell button) and re-open the console.
 
-### For a given user only one shell can be active
-  - **Details**: If the user launches Bash shell first and shortly opens PowerShell, it would connect back to the Bash shell with blue background.
-  Similarly, if the user launches PowerShell first and shortly opens Bash shell, it would connect back to PowerShell instance with black background.
-  - **Workaround**: Restart the shell by clicking [Restart Cloud Shell](media/shell-recycle.png) in the shell IFrame (refreshing the browser tab will not work).
+### For a given user, only one shell can be active
+  - **Details**: Users can only launch one of the shell, either [Bash](https://portal.azure.com) or [PowerShell](https://aka.ms/PSCloudPreview).
```
Comment (mirichmo) [inline docs/limitations.md]: "Users can only launch one type of shell at a time, ..."

**Comment by HemantMahawar** [comment]
@mirichmo Thanks for your input. Comments addressed ... Can you review again?

**Pull request merged**